build_host_setup_fedora.sh

```diff
@@ -24,6 +24,7 @@ sudo dnf install -y \
     curl \
     pkgconfig \
     libicu-devel \
+    procps-ng \
```

Comment:
Is this a required dependency? Seems like it's a tool for browsing the proc file system